frame/support/procedural/src/lib.rs

```diff
@@ -818,18 +818,66 @@ pub fn config(_: TokenStream, _: TokenStream) -> TokenStream {
 /// The `#[pallet::constant]` attribute can be used to add an associated type trait bounded by `Get`
 /// from [`pallet::config`](`macro@config`) into metadata, e.g.:
 ///
-/// ```ignore
-/// #[pallet::config]
-/// pub trait Config: frame_system::Config {
-/// 	#[pallet::constant]
-/// 	type Foo: Get<u32>;
+/// ## Example:
+///
+/// ```
+/// #[frame_support::pallet]
+/// mod pallet {
+///     use frame_support::pallet_prelude::*;
+///
+///     # #[pallet::pallet]
+///     # pub struct Pallet<T>(_);
+///
+///     #[pallet::config]
+///     pub trait Config: frame_system::Config {
+/// 		/// This is like a normal `Get` trait, but it will be added into metadata.
+/// 		#[pallet::constant]
+/// 		type Foo: Get<u32>;
+/// 	}
 /// }
 /// ```
 #[proc_macro_attribute]
 pub fn constant(_: TokenStream, _: TokenStream) -> TokenStream {
 	pallet_macro_stub()
 }
 
+/// Declares an implementation block to be the dispatchable portion of a pallet. In other words,
```

Comment:
```suggestion
/// Declares an implementation block to be the callable portion of a pallet. In other words,
```

I see below that the trait `Dispatchable` shows up, but my initial intuition was to leave this word out of the docs. People understand callable. Dispatchable less so.